drf_excel/renderers.py

```diff
@@ -56,12 +56,12 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
         """
         Render `data` into XLSX workbook, returning a workbook.
         """
-        if not self._check_validation_data(data):
-            return json.dumps(data)
-
         if data is None:
             return bytes()
```

Comment:
This code was unreachable,  `self._check_validation_data(data)` raises an error if `data` is None